lib/macho/macho_file.rb

```diff
@@ -556,7 +556,7 @@ def low_fileoff
 
       segments.each do |seg|
         sections(seg).each do |sect|
-          next if sect.size == 0
+          next if sect.size.zero?
```

Comment:
:+1:

> - Put a `# rubocop:disable Style/ZeroLengthPredicate` comment at the end of the line to locally silence the misguided advice.

I was still on RuboCop 0.41.2. That's why I didn't notice this was now causing multiple complaints and this suggestion isn't exactly practical.